web/template/layout/fee_reduction.gohtml

```diff
@@ -0,0 +1,55 @@
+{{ define "fee-reduction" }}
+    <details class="govuk-details" data-module="govuk-details">
+        <summary class="govuk-details__summary">
+                    <span class="govuk-details__summary-text" id="f-fee-reductions-tab">
+                        Fee reductions
+                    </span>
+        </summary>
+        <div class="govuk-details__text">
+            {{ range $i, $p := .FeeReductions }}
+                <table class="govuk-table table__no-border">
+                    <tbody class="govuk-table__body">
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Reduction type:</th>
+                        <td class="govuk-table__cell">
+                            <strong>
+                                {{ range $.FeeReductionTypes }}
+                                    {{ if eq .Handle $p.FeeReductionType }}{{ .Label }}{{ end }}
+                                {{ end }}
+                            </strong>
+                        </td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Date reduction approved:</th>
+                        <td class="govuk-table__cell">{{ formatDate .PaymentDate }}</td>
+                    </tr>
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Evidence:</th>
+                        <td class="govuk-table__cell" style="white-space:pre-wrap;">{{ .PaymentEvidence }}</td>
```

Comment:
I initially wanted to use a function to format this but that threw up some cross site scripting warnings and i found this in line css style the simplest solution even though i generally dislike in-line styling 

The function in question was:

Reply:
Probably sensible to move it into an `app-...` class to avoid the inline-ness and make it reusable.